core/themes/basis/template.php

```diff
@@ -53,6 +68,16 @@ function basis_preprocess_page(&$variables) {
   }
 }
 
+/**
+ * Every time we add supplemental CSS we add the version number here
+ * and update the default config file in Basis.
+ */
+
+// TODO: Before merging, we need to reset these values.
+function basis_css_versions_updated() {
+  return array('1.31.5', '1.30', '1.28.4', '1.23', 'No updates');
```

Comment:
Let's take out 'No updates' here and move to append it in the form only.
```suggestion
  return array('1.31.5', '1.30', '1.28.4', '1.23');
```